docs/developer/local_dev.md

```diff
@@ -24,5 +24,5 @@
 3. Deploy OADP operator using your image:
 
     ```
-    IMG=quay.io/<CONTAINER_REGISTRY_USERNAME>/oadp-operator:<IMAGE_TAG> make deploy
+    IMG=quay.io/<CONTAINER_REGISTRY_USERNAME>/oadp-operator:<IMAGE_TAG> make docker-push
```

Comment:
I would actually delete this file in favor of install_from_source.md

docker-push here won't be doing much help with creating an installable bundle.

Reply:
so should I just nuke developer/local_dev.md ?

Reply:
That would be preferable for me.

Reply:
done. thanks!